tests/API/CPE/lang/test_api_cpe_lang.c

```diff
@@ -266,7 +230,6 @@ void print_usage(const char *program_name, FILE * out)
 		"  %s --set-all CPE_LANG_XML ENCODING NS_PREFIX NS_HREF (PLATFORM_ID)*\n"
 		"  %s --set-key CPE_LANG_XML ENCODING KEY ID (TITLE)*\n"
 		"  %s --set-new CPE_LANG_XML ENCODING NS_PREFIX NS_HREF (PLATFORM_ID)*\n"
-		"  %s --match-cpe CPE_LANG_XML ENCODING CPE_NAME1 CPE_NAME2\n"
 		"  %s --smoke-test\n",
 		program_name, program_name, program_name, program_name,
 		program_name, program_name, program_name, program_name);
```

Comment:
When you drop a line above in the `fprintf`, you should also drop a `program_name` argument. :)

```c
[237/321] Building C object tests/API/CPE/lang/CMakeFiles/test_api_cpe_lang.dir/test_api_cpe_lang.c.o
../tests/API/CPE/lang/test_api_cpe_lang.c: In function ‘print_usage’:
../tests/API/CPE/lang/test_api_cpe_lang.c:226:3: warning: too many arguments for format [-Wformat-extra-args]
   "Usage: \n\n"
   ^~~~~~~~~~~~~
```